File: jung/src/main/java/Enron.java
public class Enron extends Core {

    private static final String PATH = "../samples/Email-Enron.txt";
    private static final Boolean AVOID_DOUBLES = true;
    private static final Integer[] UNIQUE_TESTS = {100, 1000, 10000};
    private static final Integer[] INDEXES = {100, 1000, 10000};

    public static void main(String[] args) {
        run(PATH, AVOID_DOUBLES, UNIQUE_TESTS, INDEXES);
    }
}

File: quickgraph/Enron.cs
namespace quickgraph
{
	class Enron : Core
	{
		public Enron()
		{
			path = "../../../samples/Email-Enron.txt";
			avoidDoubles = true;
			uniqueTests = new int[] { 100, 1000, 10000 };
			indexes = new int[] { 100, 1000, 10000 };
			algorithms = new string[] { "bfs", "dfs", "shortest" };
		}
	}
}